<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointPropertiesDlg</class>
 <widget class="QDialog" name="PointPropertiesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Points Properties</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QToolButton" name="markPointButton">
     <property name="toolTip">
      <string comment="Mark point">Select one point and mark it with some information</string>
     </property>
     <property name="statusTip">
      <string comment="Mark point">Select one point and mark it with some information</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/MarkObject/images/moMarkPoint.png</normaloff>:/MarkObject/images/moMarkPoint.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="markLineButton">
     <property name="toolTip">
      <string comment="Mark line">Select a series of points through a line and mark the line with some information</string>
     </property>
     <property name="statusTip">
      <string comment="Mark line">Select a series of points through a line and mark the line with some information</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/MarkObject/images/moMarkLine.png</normaloff>:/MarkObject/images/moMarkLine.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="markAreaButton">
     <property name="toolTip">
      <string comment="Mark area">Select a series of points through a circle and mark the area in the circle with some information</string>
     </property>
     <property name="statusTip">
      <string comment="Mark area">Select a series of points through a circle and mark the area in the circle with some information</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/MarkObject/images/moMarkArea.png</normaloff>:/MarkObject/images/moMarkArea.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="markObjectBagButton">
     <property name="toolTip">
      <string comment="Mark object bag">Select a series of objects in a bag</string>
     </property>
     <property name="statusTip">
      <string comment="Mark object bag">Select a series of objects in a bag</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/MarkObject/images/moMarkObjectBag.png</normaloff>:/MarkObject/images/moMarkObjectBag.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="markUndoButton">
     <property name="toolTip">
      <string comment="Mark undo">Undo mark operation</string>
     </property>
     <property name="statusTip">
      <string comment="Mark undo">Undo mark operation</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/MarkObject/images/moUndo.png</normaloff>:/MarkObject/images/moUndo.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="markRedoButton">
     <property name="toolTip">
      <string comment="Mark redo">Redo mark operation</string>
     </property>
     <property name="statusTip">
      <string comment="Mark redo">Redo mark operation</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/MarkObject/images/moRedo.png</normaloff>:/MarkObject/images/moRedo.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="markDoneButton">
     <property name="toolTip">
      <string comment="Mark done">Finish mark operation of a line or an area</string>
     </property>
     <property name="statusTip">
      <string comment="Mark done">Finish mark operation of a line or an area</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/MarkObject/images/moDone.png</normaloff>:/MarkObject/images/moDone.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pointPropertiesButton">
     <property name="toolTip">
      <string comment="Display selected point properties">Select one point and display its information</string>
     </property>
     <property name="statusTip">
      <string comment="Display selected point properties">Select one point and display its information</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/CC/images/smallPointProperties.png</normaloff>:/CC/images/smallPointProperties.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pointPointDistanceButton">
     <property name="toolTip">
      <string comment="Compute point to point distance">Select 2 points and display segment information (length, etc.)</string>
     </property>
     <property name="statusTip">
      <string comment="Compute point to point distance">Select 2 points and display segment information (length, etc.)</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/CC/images/smallPointDistance.png</normaloff>:/CC/images/smallPointDistance.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pointsAngleButton">
     <property name="toolTip">
      <string>Select 3 points and display corresponding triangle information</string>
     </property>
     <property name="statusTip">
      <string>Select 3 points and display corresponding triangle information</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/CC/images/smallPointsAngle.png</normaloff>:/CC/images/smallPointsAngle.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="rectZoneToolButton">
     <property name="toolTip">
      <string>Define a rectangular 2D label</string>
     </property>
     <property name="statusTip">
      <string>Define a rectangular 2D label</string>
     </property>
     <property name="text">
      <string>2D zone</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/CC/images/smallRectangleSelect.png</normaloff>:/CC/images/smallRectangleSelect.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="saveLabelButton">
     <property name="toolTip">
      <string>Save current label (added to cloud children)</string>
     </property>
     <property name="statusTip">
      <string>Save current label (added to cloud children)</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/CC/images/ccSave.png</normaloff>:/CC/images/ccSave.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="razButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/CC/images/smallReset.png</normaloff>:/CC/images/smallReset.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="closeButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/CC/images/smallCancel.png</normaloff>:/CC/images/smallCancel.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
